miv/visualization/spike_triggered_avg.py

```diff
@@ -0,0 +1,66 @@
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from miv.statistics.spiketrain_statistics import binned_spiketrain
+from miv.typing import SignalType, SpikestampsType
+
+
+def spike_triggered_avg(
+    signal: SignalType,
+    channel_x: float,
+    spiketrains: SpikestampsType,
+    channel_y: float,
+    sampling_freq: float,
+    win: float,
```

Comment:
```suggestion
    window_length: float,
```

It is typically more friendly to have explicit name for parameters.

Reply:
38fe9b3ceef9335122c03681ce4201d001189734